packages/desktop-client/src/components/accounts/Account.tsx

```diff
@@ -1357,6 +1358,18 @@ class AccountInternal extends PureComponent<
 
   onApplyFilter = async (conditionOrSavedFilter: ConditionEntity) => {
     let filterConditions = this.state.filterConditions;
+
+    if (
```

Comment:
TBH I'm not entirely certain why we need this logic. But I have now brought it back to verbatim match what we had before (except for a few TS patches): https://github.com/actualbudget/actual/commit/5cfa2cf577d94a99ddc55b519847fd79547e881b#diff-99c276500045bfda8f9c14b3c10705730a1fe7452a69246d4e40d092c24d305eL1199